src/test/java/org/example/TicketServiceTest.java

```diff
@@ -0,0 +1,125 @@
+package org.example;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.sql.Date;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+public class TicketServiceTest {
+
+    private TicketService ticketService;
+
+    private TicketDAO ticketDAO;
+
+    @BeforeEach
+    void setUp() {
+        ticketDAO = Mockito.mock(TicketDAO.class);
+        ticketService = new TicketService(ticketDAO);
+    }
+
+    @Test
+    void findTicket() {
```

Comment:
name should be more self descriptive and you should do more granular tests. This one should be split for case when ticket actually returned and when we could not find ticket by id